misc/assets/inforequest-flow.svg

```diff
@@ -0,0 +1,343 @@
+    </foreignObject>
+    <path class="arrow" d="M 0,-30 v -20 h 130 l 20,20" />
+  </g>
+
+  <g transform="translate(450 670)">
+    <path class="arrow" d="M 150,0 h -50" />
+    <use href="#action" />
+    <foreignObject x="-100" y="-30" width="200" height="60">
+      <xhtml:div class="action">
+        <xhtml:div class="caption">Full Disclosure</xhtml:div>
+      </xhtml:div>
+    </foreignObject>
+    <path class="arrow" d="M -100,0 h -50 v -50 h 150 v 20" />
+    <path class="arrow" d="M -100,0 h -155" />
+  </g>
+
+  <g transform="translate(175 670)">
+    <use href="#end" />
+  </g>
+
+  <g transform="translate(450 770)">
+    <path class="arrow" d="M 150,0 h -50" />
+    <use href="#action" />
+    <foreignObject x="-100" y="-30" width="200" height="60">
+      <xhtml:div class="action">
+        <xhtml:div class="caption">Non-full Discl.</xhtml:div>
+        <xhtml:div class="note">applicant: 15 CD no SZ</xhtml:div>
+        <xhtml:div class="note">after: DD</xhtml:div>
+      </xhtml:div>
+    </foreignObject>
+    <path class="arrow" d="M -100,0 h -50 v -50 h 150 v 20" />
+    <path class="arrow" d="M -100,0 h -230 l -20,20" />
+    <g transform="translate(-330 5)">
+      <text class="label">
+        <tspan>at most 47 CD</tspan>
+        <tspan x="0" dy="1.2em">after DL</tspan>
+      </text>
+    </g>
+  </g>
+
+  <g transform="translate(450 870)">
+    <path class="arrow" d="M 150,0 h -50" />
+    <use href="#action" />
+    <foreignObject x="-100" y="-30" width="200" height="60">
+      <xhtml:div class="action">
+        <xhtml:div class="caption">Refusal</xhtml:div>
+        <xhtml:div class="note">applicant: 15 CD no SZ</xhtml:div>
+        <xhtml:div class="note">after: DD</xhtml:div>
+      </xhtml:div>
+    </foreignObject>
+    <path class="arrow" d="M -100,0 h -50 v -50 h 150 v 20" />
+    <path class="arrow" d="M -100,0 h -230 l -20,20" />
+    <g transform="translate(-330 5)">
+      <text class="label">
+        <tspan>at most 7 CD</tspan>
+        <tspan x="0" dy="1.2em">after DL</tspan>
+      </text>
+    </g>
+  </g>
+
+  <g transform="translate(450 970)">
+    <path class="arrow" d="M 150,-830 v 830 h -50" />
+    <use href="#action" />
+    <foreignObject x="-100" y="-30" width="200" height="60">
+      <xhtml:div class="action">
+        <xhtml:div class="caption">Expiration</xhtml:div>
+        <xhtml:div class="note">applicant: 15 CD no SZ</xhtml:div>
+        <xhtml:div class="note">after: LD</xhtml:div>
+      </xhtml:div>
+    </foreignObject>
+    <path class="arrow" d="M -100,0 h -230 l -20,20" />
+    <g transform="translate(-330 5)">
+      <text class="label">
+        <tspan>at most 47 CD</tspan>
+        <tspan x="0" dy="1.2em">after DL</tspan>
+      </text>
+    </g>
+  </g>
+
+  <g transform="translate(100 1070)">
+    <use href="#action" />
+    <foreignObject x="-100" y="-30" width="200" height="60">
+      <xhtml:div class="action">
+        <xhtml:div class="caption">Appeal</xhtml:div>
```

Comment:
V `inforequest-flow.odg` bolo zo stavu `Appeal` (resp. `Advancement`) prechod este do stavu "prazdne prerusovany kruh".

Reply:
@viliambalaz Aha. Hej, ten prazdny kruzok mal znamenat, ze fyzicky to odvolanie "prejde rukami" institucie (obligee) a az ta ho posunie nadradenej institucii (obligee superior). Ale vyhodil som ho odtial, kedze relane to nie je skutocna akcia z nasho pohladu. Ani ju nijak v DB nemame zaznamenanu. Takze som v nakrese nechal iba ciaru z odvolania ku akciam robenym nadradenou instituciou.